src/client/pages/tocheckitem.react.js

```diff
@@ -0,0 +1,21 @@
+import Component from '../components/component.react';
+import React from 'react';
+import {FormattedHTMLMessage} from 'react-intl';
+
+class ToCheckItem extends Component {
+
+  render() {
+    return (
+      <li>
+        <FormattedHTMLMessage message={this.props.message} />
+      </li>
+    );
+  }
+
+}
+
+ToCheckItem.propTypes = {
+  message: React.PropTypes.string.isRequired
```

Comment:
I don't think message has to be passed, when it's actually constant. Do you agree?